tox.ini

```diff
@@ -1,6 +1,7 @@
 [tox]
 envlist =
-    py{38}-dj{22,30}-wagtail{28,master}
+    py310-dj32-wagtaillts
```

Comment:
question: since the package officially supports more python versions, should it not test against older ones too? 😈 

I don't have an issue with this per se, but packages usually will test against several recent ones

Reply:
Done it!